facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -219,6 +219,16 @@ private boolean shouldCheckConsistency(FacadeOperation facadeOperation) {
         }
     }
 
+    private void cancel(ListenableFuture listenableFuture) {
+        if (listenableFuture != null) {
+            try {
+                listenableFuture.cancel(true);
+            } catch (Exception e) {
+                log.error("Failed to cancel lightblue call", e);
```

Comment:
You mean a debug statement that it couldn't be cancelled? I don't think it calls for a warning, error even more so.